src/Components/RelatedCollectionsRail/RelatedCollectionEntity.tsx

```diff
@@ -45,12 +46,13 @@ export class RelatedCollectionEntity extends React.Component<CollectionProps> {
           href={`${sd.APP_URL}/collection/${slug}`}
           onClick={this.onLinkClick.bind(this)}
         >
+          Hey!
```

Comment:
This looks like a test message that snuck into the commit.

Reply:
New brand guidelines. Must be excited! in! every! component!

![heyy](https://user-images.githubusercontent.com/140521/65970088-de90b000-e433-11e9-9805-060bd6493535.png)

(Lol ok this is fixed now 🙈 )